cypress-tests/cypress/e2e/PaymentUtils/Commons.js

```diff
@@ -149,9 +149,7 @@ with `getCustomExchange`, if 501 response is expected, there is no need to pass
 
 // Const to get default PaymentExchange object
 const getDefaultExchange = () => ({
-  Request: {
-    currency: "EUR",
-  },
+  Request: {},
```

Comment:
what will be currency by default? USD?

Reply:
This was previously added since we mainly used it for bank redirects which is commonly used in the EU.  

We recently changed the focus to use `getCustomExchange()` for customisation of requests. And this resulted in some connectors like Stripe to have issues during NTID flow where currency was accidentally being passed in the confirm call which was unintended. 

By default, we do not pass any currency. Since `PaymentIntent` is passed in the initial call, whatever the currency that is passed in there will be sent in the request and `USD` is used in common